opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/README.adoc

```diff
@@ -0,0 +1,479 @@
+== OpenNMS RESTv2 Interface
```

Comment:
I put it inside the REST package because the API is still incomplete and may undergo more changes. I'd rather get the node endpoints done and tested before we make it a fully public API.